modules/ROOT/pages/oauth-policy-implementation-concept.adoc

```diff
@@ -12,55 +12,66 @@ image::oauth-implementation-federated.png[]
 . The federated policy validates the token against the OAuth provider. To improve performance, Mule caches the call to the authorization server, which is only performed once during the lifetime of the token.
 +
 When OpenId, PingFederate, or OAuth 2.0 Access Token Enforcement Using External Provider policies are in effect and your API is deployed under Mule 4, the client ID returned by the OAuth provider is validated by the Client Id Enforcement policy.
-. All fulfilled requests in the flow are enriched with the information provided by the OAuth Provider when running under Mule 4, but information is not returned to the user. In releases earlier than Mule 4, this information was returned to the user.
+. As opposed to Mule 3, in Mule 4, all fulfilled requests in the flow are enriched with the information provided by the OAuth Provider, but this information is not returned to the user.
 
```

Comment:
Unless explicitly configured in the policy by setting the "Expose Headers" checkbox. 

<img width="772" alt="Screen Shot 2019-04-01 at 2 06 06 PM" src="https://user-images.githubusercontent.com/1156112/55345909-7ca4bd00-5487-11e9-8326-765eabea1b05.png">

Reply:
Thanks! I assume this would look something like this: 16541f4 ?

Reply:
yes! looks good now, thanks for making the change